bool start
---
int32[] aruco_ids
geometry_msgs/Pose[] aruco_poses
---
int32 remaining_ids